src/dialogsearch.cpp

```diff
@@ -21,6 +22,13 @@ DialogSearch::DialogSearch(QWidget *parent) :
     ui->list_result->setSelectionBehavior(QAbstractItemView::SelectRows);
     ui->list_result->setEditTriggers(QAbstractItemView::NoEditTriggers);
 
+
+    for (auto *box : {ui->combo_variable, ui->combo_switch, ui->combo_item, ui->combo_eventname})
+    {
+        box->completer()->setFilterMode(Qt::MatchContains);
+        box->completer()->setCompletionMode(QCompleter::PopupCompletion);
+    }
+
```

Comment:
Do you see any (future) case here where auto-complete is unwanted in some comboboxes?

If not you could use QObject::findChildren<QComboBox*>() here (untested!)